check_is_number_prime.py

```diff
@@ -0,0 +1,39 @@
+def get_number():
+    while True:
+        try:
+            number = int(input('\nВведите натуральное число: '))
+            if number == 0 or number == 1:
+                print('Это число не является ни простым, ни составным')
+            elif number < 0:
+                print('Простое число может быть только натуральным.')
+                print('То есть необходимо ввести целое, положительное число.')
+            else:
+                 return number
+        except ValueError:
```

Comment:
It's better to cover with `try...except` only code that can create an error, the rest of the code can be after `except`

Reply:
I did that, but it worked bad. If except instruction worked out the progtam return me None and mistake after that.